### ISSUE TITLE
Poor color contrast on Alert

### ISSUE THREAD

**Issue by bmuenzenmeyer** [nodejs/nodejs.org#8290]
<img width="728" height="136" alt="Image" src="https://github.com/user-attachments/assets/a64b495b-300d-4f53-bbd9-af27e965a5bf" />

Example https://nodejs.org/en/blog/migrations/v20-to-v22

**Comment by bmuenzenmeyer** [comment]
odd - it seems correct in the storybook https://www.chromatic.com/component?appId=64c7d71358830e9105808652&csfId=src-common-alertbox&buildNumber=3628&k=6901f2aa232fcc1b0157da01-light+desktop-interactive-true&h=5&b=4 (i think thats the right URL, but it was from my history so maybe not)

**Comment by avivkeller** [comment]
I faced a similar issue during the development of doc-kit. It's because there's an extra child element:

<img width="134" height="135" alt="Image" src="https://github.com/user-attachments/assets/fc80a22c-148e-44b0-a37d-dee4019115f7" />

(i.e. it should be the component and then the text, not the component with the text wrapped)

I'm in favor of removing the AlertBox from the MD entirely, and just putting that as plaintext in the blogs

**Comment by bmuenzenmeyer** [comment]
ah yes, that happened somewhere else too

**Comment by JakobJingleheimer** [comment]
There's an experimental built-in function for this now: [`contrast-color()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/contrast-color)

**Comment by AugustinMauroy** [comment]
It's an issue bettewn component style and markdown paragraph style. For strange reason the parapha style take over the component style.

**Comment by Ju-MINJAE** [comment]
can i fix it?

**Comment by AugustinMauroy** [comment]
> can i fix it?

Yeah, feel free to open an issue
